osbrain/agent.py

```diff
@@ -1219,7 +1217,7 @@ def shutdown(self):
 
     def kill(self):
         self.close_sockets()
-        self.kill_agent = True
+        self._pyroDaemon.shutdown()
```

Comment:
It works... Why? I can't find where we are setting Agent._pyroDaemon, so my guess is that it works simply because we are always supposed to access the agent/nameserver through a proxy or something like that? And even in that case, I don't understand how it doesn't raise an `AttributeError`. Could you explain a bit please? :)